content/en/flux/security/slsa-assessment.md

```diff
@@ -0,0 +1,188 @@
+
+### Build Platform Requirements
+
+#### Provenance generation
+
+| Requirement               | Required at SLSA L3 | Met by Flux |
+|---------------------------|---------------------|-------------|
+| Provenance Exists         | Yes                 | Yes         |
+| Provenance is Authentic   | Yes                 | Yes         |
+| Provenance is Unforgeable | Yes                 | Yes         |
+
+> The build process MUST generate provenance that unambiguously identifies the output package by cryptographic digest
+> and describes how that package was produced.
+
+- The Flux project release workflows make use of the
+  official [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator) for provenance
+  generation.
+- The provenance file stores the SHA-256 hashes of the release artifacts (binaries, SBOMs, deploy manifests, source
+  code).
+- The provenance identifies the Flux container images using their digest in SHA-256 format.
+
+> Consumers MUST be able to validate the authenticity of the provenance attestation in order to ensure integrity and
+> define trust.
+
+- The provenance is signed by Sigstore Cosign using the GitHub OIDC identity and the public key to verify the provenance
+  is stored in the public [Rekor transparency log](https://docs.sigstore.dev/rekor/overview/).
+- The release process and the provenance generation runs in isolation on an ephemeral environment provided by
+  GitHub-hosted runners.
+- The provenance of the Flux release artifacts (binaries, container images, SBOMs, deploy manifests) can be verified
+  using the official [SLSA verifier tool](https://github.com/slsa-framework/slsa-verifier).
+
+> Provenance MUST be strongly resistant to forgery by tenants.
+
+- The provenance generation workflows run on ephemeral and isolated virtual machines which are fully managed by GitHub.
+- The provenance signing secrets are ephemeral and are generated through
+  Sigstore's [keyless signing](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) procedure.
+- The [SLSA GitHub generator](https://github.com/slsa-framework/slsa-github-generator) runs on separate virtual machines
+  than the build and release process, so that the Flux build scripts don't have access to the signing secrets.
+
+#### Isolation strength
+
+| Requirement | Required at SLSA L3 | Met by Flux |
+|-------------|---------------------|-------------|
+| Hosted      | Yes                 | Yes         |
+| Isolated    | Yes                 | Yes         |
+
+> All build steps ran using a hosted build platform on shared or dedicated infrastructure.
+
+- The release process and the provenance generation runs in isolation on an ephemeral environment provided by
+  GitHub-hosted runners.
+- The provenance generation is decoupled from the build process,
+  the [SLSA GitHub generator](https://github.com/slsa-framework/slsa-github-generator) runs on separate virtual machines
+  fully managed by GitHub.
```

Comment:
```suggestion
- The provenance generation for an artifact is decoupled from the build process,
  since the [SLSA GitHub generator](https://github.com/slsa-framework/slsa-github-generator) runs on a separate virtual machine
  fully managed by GitHub.
```

Reply:
I don't think this is OK, it's not one artifact, there are many.